packages/formation/jest.config.js

```diff
@@ -1,7 +1,8 @@
 module.exports = {
   transform: {
-    "^.+\\.js$": "<rootDir>/packages/formation/scripts/babel-transformer.js",
+    '^.+\\.js$': '<rootDir>/packages/formation/scripts/babel-transformer.js',
   },
+  testEnvironment: 'jsdom',
```

Comment:
This fixes the issue with `window` not being defined. The default environment is `node`, where things like `window` don't exist.

https://jestjs.io/docs/configuration#testenvironment-string